onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -852,6 +852,15 @@ def aten_asinh(self: TFloat) -> TFloat:
     return op.Asinh(self)
 
 
+@torch_op("aten::_assert_async.msg")
+def aten_assert_async(
+    self: TTensor, assert_msg: str = ""
+) -> TTensor:  # pylint: disable=unused-argument`
+    """_assert_async.msg(Tensor self, str assert_msg) -> ()"""
```

Comment:
Signature says this function does not have an output

Reply:
hmmm maybe we should delete it.